docs/querying/basics.md

```diff
@@ -109,8 +109,17 @@ single sample value for each at a given timestamp (point in time).  In the simpl
 form, only a metric name is specified, which results in an instant vector
 containing elements for all time series that have this metric name.
 
+The value returned will be that of the most recent sample prior to the query's
+evaluation timestamp (in the case of an
+[instant query](api.md#instant-queries))
+or of the current resolution step within the query (in the case of a
+[range query](api.md/#range-queries)).
+The [`@` modifier](#modifier) allows overriding the timestamp relative to which
+the selection takes place. Time series where the most recent sample is older
+than the [staleness threshold](#staleness) will not be returned.
```

Comment:
```suggestion
the selection takes place. Time series are only returned if their most recent sample is less than the [lookback period](#staleness) ago.
```

This correctly defines the behavior in v3 (which is already merged into main, so we should let the documentation refer to it).